src/main/java/de/teamlapen/vampirism/command/EyeCommand.java

```diff
@@ -27,9 +28,7 @@
 
     private static int setEye(CommandContext<CommandSource> context, PlayerEntity player, int type) {
         if (VampirePlayer.get(player).setEyeType(type)) {
-            context.getSource().sendFeedback(new TranslationTextComponent("command.vampirism.base.eye.success", type), true);
-        } else {
-            context.getSource().sendErrorMessage(new TranslationTextComponent("command.vampirism.base.eye.types", REFERENCE.EYE_TYPE_COUNT - 1));
+            context.getSource().sendFeedback(new TranslationTextComponent("command.vampirism.base.eye.success", type), false);
```

Comment:
I assume an error is given somewhere else?

Reply:
The IntegerArgumentType only allows 0 - REFERENCE.EYE_TYPE_COUNT -1, otherwise throws an error